tests/pull.bats

```diff
@@ -92,15 +92,8 @@ load helpers
 }
 
 @test "pull-from-docker-daemon" {
-  run systemctl status docker
-  if [[ ! "$output" =~ "active (running)" ]]
-  then
-     skip "Skip the test as docker services is not running"
-  fi
+  skip_if_no_docker
 
-  run systemctl start docker
-  echo "$output"
-  [ "$status" -eq 0 ]
```

Comment:
Judgment call: removed, because we've literally just checked that the service is active, and `start` != `restart`. If there is some hidden side effect or reason for running this `start`, please speak now.